update/generate.groovy

```diff
@@ -4,7 +4,7 @@ import groovy.json.JsonOutput
 wantedPlugins = [:]
 resultingPlugins = [:]
 updateCenter = null
-debug = false
+debug = true
```

Comment:
Do you want to keep debug on?

Reply:
I'll check. I think it is more "verbose" than "debug"

Reply:
changed to "verbose", kept true